src/Interpreters/Streaming/Aggregator.cpp

```diff
@@ -187,7 +190,7 @@ void AggregatedDataVariants::reset()
     switch (type)
     {
         case AggregatedDataVariants::Type::EMPTY:       break;
-        case AggregatedDataVariants::Type::without_key: break;
+        case AggregatedDataVariants::Type::without_key: without_key = nullptr; break;
```

Comment:
I think we can reset `without_key = nulllptr` blindly for all cases ?

Reply:
This is useless logic, I will delete it,  it has been cleared in destroyAllAggregateStates.